Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -634,6 +646,9 @@ IssueCategory[] GetSelectedCategories()
                 requestedCategories.AddRange(m_ProjectAuditor.GetModule<AssetsModule>().GetCategories());
             if ((m_SelectedModules.HasFlag(BuiltInModules.BuildReport)))
                 requestedCategories.AddRange(m_ProjectAuditor.GetModule<BuildReportModule>().GetCategories());
+            if ((m_SelectedModules.HasFlag(BuiltInModules.InstalledPackages)))
```

Comment:
since this module is currently "hidden" as developer/experimental and the module selection is being refactored, please remove from the Home page